polaris.shopify.com/content/patterns/date-picking.ts

```diff
@@ -31,8 +31,8 @@ const pattern: MultiVariantPattern = {
       usefulToKnow: `
 | | |
 |-|-|
-|Labels need to simply depict the task at hand. Whether that be a start date, end date, start time etc.|![](/images/patterns/single-list-usage-1.png)|
-|This pattern can be duplicated to allow users to add an end date or time.|![](/images/patterns/single-list-usage-2.png)|
+|Labels need to simply depict the task at hand. Whether that be a start date, end date, start time etc.|![Date input labeled “Expiry date”](/images/patterns/single-list-usage-1.png)|
+|This pattern can be duplicated to allow users to add an end date or time.|![“Active dates” section with “start date” and “end date” inputs, toggled on with a “Set end date” checkbox.](/images/patterns/single-list-usage-2.png)|
```

Comment:
I wonder if "toggled on" is clear. Perhaps "with a checked checkbox labeled 'Set end date'" or similar?